src/ts/index.ts

```diff
@@ -1,2 +1,5 @@
+// Import to extend Conviva types.
+import './conviva/ConvivaExtension';
```

Comment:
Would it make sense to also export the Conviva objects? Because currently I believe constants like `Conviva.Constants.DeviceCategory.WEB` (which we also show in the README) might not be accessible (or are they?).

Reply:
Users can import constants from the Conviva package:

<img width="767" alt="image" src="https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/assets/85180621/5af1b47e-1770-47d5-9ae3-bff0d80fb1be">

Reply:
Once there is import from `@bitmovin/player-integration-conviva` the extension is imported automatically as well:

<img width="936" alt="image" src="https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/assets/85180621/884d5587-eb36-4e2e-a573-b10b82e795b9">

The `StorageInterface` is from `ConvivaExtension`. So if I CMD + click on the `StorageInterface` it is resolved to:

<img width="754" alt="image" src="https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/assets/85180621/a38d38e2-7903-4482-99ff-e36ed46ace12">

NOTE: I have mapped `@bitmovin/player-integration-conviva` to a local folder for development:

<img width="912" alt="image" src="https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/assets/85180621/48592ea2-7a2e-48a8-ac93-320a2258140a">

Reply:
Ah yes, of course 🤦  The Conviva lib is just a peer dependency, so the customer anyway needs to install it and can access the imports of it. Thanks for clarifying 👍